pulsar-client-cpp/CMakeLists.txt

```diff
@@ -64,7 +62,8 @@ set(CLIENT_LIBS
   pulsar
 )
 
-message( STATUS "GCC_DUMP_VERSION: " ${GCC_DUMP_VERSION} )
+message( STATUS "CMAKE_C_COMPILER: " ${CMAKE_C_COMPILER} )
```

Comment:
Cmake is already printing the gcc / g++ versions. It prints it only the first time when it does the configuration phase.

If you remove, `CMakeCache` and do `cmake .` again, you'll see it printed